_pages/install/integrations/opf-integration/opf-child-pages/2-quickbuy.md

```diff
@@ -0,0 +1,134 @@
+
+## Enabling Quick Buy
+
+You can enable quick buy by installing the `@spartacus/opf` OPF library. For more information, see [Installing Additional Spartacus Libraries]({{ site.baseurl }}/schematics/#installing-additional-spartacus-libraries). 
+
+For a seamless installation, you can use schematics parameters. For more information, see how to use --opfGooglePayApiUrl in 'Configuring Quic kBuy for GooglePay'.
+
+### CMS Components
+
+Quick buy is CMS-driven and consists of the following CMS component:
+
+-`OpfQuickBuyButtonsComponent`
+
+You can configure quick buy by using SmartEdit to display the quick buy component in Spartacus, or you can manually add it to content slots using ImpEx.
+
+If you are using the [{% assign linkedpage = site.pages | where: "name", "spartacussampledata-extension.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/install/spartacussampledata-extension.md %}), the quick buy component is already enabled. However, if you decide not to use the `spartacussampledata` extension, you can enable the quick buy component through ImpEx.
+
+**Note:** The `$contentCV` variable that is used throughout the following ImpEx examples, and which stores information about the content catalog, is defined as follows:
+
+```text
+$contentCatalog=electronics-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Online])[default=$contentCatalog:Online]
+```
+
+### Adding CMS Component Manually
+
+This section describes how to add the quick buy CMS component to Spartacus using ImpEx.
+
+You can enable quick buy by adding the Quick Buy button component to the Cart page/You can enable the checkout components for open payment framework with the following ImpEx:
+
+```text
+INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType
+;;OpfQuickBuyButtonsComponent;Opf Quick Buy Buttons Component;OpfQuickBuyButtonsComponent
+
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;cmsComponents(uid, $contentCV)
+;;CenterRightContentSlot-cartPage;Center Right Content Slot for Cart Page;CartTotalsComponent,CartApplyCouponComponent,CartQuickOrderFormComponent,OpfQuickBuyButtonsComponent,CartProceedToCheckoutComponent
+```
+
+## Configuring Quick Buy
+
+### Configuring Quick Buy for Apple Pay
+
+The open payment framework enables you to implement an express checkout for your customers using Apple Pay.
+
+#### Browser Support
+
+Quick buy for Apple Pay is available on Safari. For more information, see https://developer.apple.com/documentation/apple_pay_on_the_web.
+
+#### Modifying the Quick Buy Button
+
+You can modify the appearance of the Apple Pay quick buy button. By default, the quick buy button is black with a 'Buy with Apple Pay' label. The default values of the Apple Pay button style are as follows:
+
```

Comment:
@FollowTheFlo @Matejk00 Hello! I know this is being picky but wanted to double check... It says here "By default, the quick buy button is black with a 'Buy with Apple Pay' label." I wanted to see where this label came from, and found some info in the Apple link you provided -- but -- in there, I could only find "_A button with the text “Buy with” and the Apple Pay logo_". I didn't see a version of the button with a label that was all text, such as "Buy with Apple Pay". 

Why am I asking? Because in the original doc input, it said "'Buy with ApplePay' label", and then Faith changed it to "'Buy with Apple Pay' label", so I wanted to find this label... And this is where I got to 😄

Reply:
Okay, I see in `%cx-opf-apple-pay` the setting `-apple-pay-button-type: buy;` .... that answers my question. Sorry for the trouble 😊